node_modules/.gitignore

```diff
@@ -5,7 +5,6 @@
 !.gitignore
 
 # less for less -> css
-!.bin/
```

Comment:
@pmclanahan - this seems to create un-tracked changes in `node_modules/.bin/` when I do a fresh `npm install`. The line below should be enough to cover the `node_modules/.bin/lockdown` files that are already checked into the repo, unless I'm missing something?